File: Android/BottomSheet/app/src/main/java/tokyo/randx/portfolio/android/bottomsheet/MainActivity.java
package tokyo.randx.portfolio.android.bottomsheet;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        init();
    }

    private void init() {
        Button btnShow = findViewById(R.id.btn_show);
        btnShow.setOnClickListener(view -> {
            ModalBottomSheet modalBottomSheet = new ModalBottomSheet();
            modalBottomSheet.show(getSupportFragmentManager(), ModalBottomSheet.TAG);
        });
    }
}

File: Android/ComposeInViews/app/src/main/java/tokyo/randx/portfolio/android/composeinviews/MainActivity.kt
package tokyo.randx.portfolio.android.composeinviews

import android.content.Intent
import android.os.Bundle
import android.widget.TextView
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        setContentView(R.layout.activity_main)
        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main)) { v, insets ->
            val systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars())
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom)
            insets
        }

        val full = this.findViewById<TextView>(R.id.full)
        full.setOnClickListener {
            val intent = Intent(this, FullComposeActivity::class.java)
            startActivity(intent)
        }

        val partial = this.findViewById<TextView>(R.id.partial)
        partial.setOnClickListener {
            val intent = Intent(this, PartialComposeActivity::class.java)
            startActivity(intent)
        }
    }
}